roles/build-release/tasks/build.yaml

```diff
@@ -68,14 +69,14 @@
   changed_when: false
   register: _tarball_checksum
 
-- name: Compute a galaxy-requirements.yml file for the release
-  command: >-
-    poetry run {{ role_path }}/files/deps-to-galaxy.py --depsfile {{ antsibull_data_dir }}/{{ _deps_file }}
+- name: Read galaxy-requirements.yml file for the release
+  ansible.builtin.slurp:
+    src: '{{ antsibull_data_dir }}/{{ _galaxy_file }}'
   register: _galaxy_reqs
 
 - name: Template a galaxy-requirements.yaml file
   copy:
     dest: "{{ antsibull_data_dir }}/galaxy-requirements.yaml"
     content: |
-      # Generated by antsibull/roles/build-release/files/deps-to-galaxy.py from {{ _deps_file }}
-      {{ _galaxy_reqs.stdout }}
+      # Collections included in Ansible {{ antsibull_ansible_version }}
+      {{ _galaxy_reqs.content | ansible.builtin.b64decode }}
```

Comment:
I thought about whether this should be a symlink now that we'd have versioned requirement files but the purpose of the file was to be a single URL that people could use and stay up to date. Thanks for keeping it :)